mkdocs.yml

```diff
@@ -288,6 +288,8 @@ pages:
       - Overview: agents/index.md
       - Sematext Agent:
         - Overview: agents/sematext-agent/index.md
+        - Release Notes: agents/sematext-agent/platform-support-policy.md
```

Comment:
Release Notes ==> Platform Support Policy @cpipilas ?

Reply:
> Release Notes ==> Platform Support Policy @cpipilas ?

New PR soon to be available: https://github.com/sematext/docs/pull/592